mail/notmuch/Portfile

```diff
@@ -5,8 +5,7 @@ PortGroup           compiler_blacklist_versions 1.0
 PortGroup           conflicts_build             1.0
 
 name                notmuch
-version             0.29.1
-revision            1
```

Comment:
We now prefer to keep the `revision` line, but set it back to 0.

Reply:
We do? https://guide.macports.org/#reference.keywords does not mention this, `revision` is documented as optional.

Reply:
There was a discussion on the mailing list about it not too long ago. The guide is behind on this.

Reply:
I try to follow the ML closely, but I don't remember this being discussed. Would you perhaps be able to point me to the relevant thread? I actually use `notmuch` for my mail, and it is pretty good at searching, but I could not find the relevant emails.